btc-withdrawal/src/main/kotlin/withdrawal/btc/transaction/TransactionHelper.kt

```diff
@@ -111,6 +111,7 @@ class TransactionHelper(
      * @param availableAddresses - set of addresses which transactions will be available to spend
      * @param wallet - current wallet. Used to fetch unspents
      * @param amount - amount of SAT to spend
+     * @param fee - tx fee that depends on inputs and outputs. Initial value is zero.
```

Comment:
Can we use default value then?

Reply:
No

Reply:
Hmm. Yes, we can do it. Thank you.

Reply:
So maybe we should split the function, one will be for internal use(this one), and other will call internal, because this function will kind of calculate optimal and necessary fee amount, than I should I put zero as a param

Reply:
Already solved